pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -68,10 +69,11 @@ class LightningKokkos(LightningQubit):
 
     name = "PennyLane plugin for Kokkos-backed Lightning device"
     short_name = "lightning.kokkos"
-    pennylane_requires = ">=0.22"
+    pennylane_requires = "<=0.28"
```

Comment:
Is this fixed yet in another PR?

Reply:
not yet.

Reply:
Fixed in this[ PR](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/31).